lib/content-services/src/lib/search/components/search.component.ts

```diff
@@ -104,31 +104,23 @@ export class SearchComponent implements SearchComponentInterface, AfterContentIn
     _classList: { [key: string]: boolean } = {};
     private onDestroy$ = new Subject<boolean>();
 
-    constructor(private searchService: SearchService,
-                private _elementRef: ElementRef) {
-        this.keyPressedStream
-            .pipe(
-                debounceTime(200),
-                takeUntil(this.onDestroy$)
-            )
-            .subscribe(searchedWord => {
-                this.loadSearchResults(searchedWord);
-            });
-
-        searchService.dataLoaded
-            .pipe(takeUntil(this.onDestroy$))
-            .subscribe(
-                nodePaging => this.onSearchDataLoaded(nodePaging),
-                error => this.onSearchDataError(error)
-            );
+    constructor(private searchService: SearchService, private _elementRef: ElementRef) {
```

Comment:
Do we have to define access modifiers just for arguments for constructor?

Reply:
what do you mean? we always make private injectors in constructors

Reply:
Ah right my bad